python/ray/runtime_env/runtime_env.py

```diff
@@ -287,6 +288,7 @@ def __init__(
         nsight: Optional[Union[str, Dict[str, str]]] = None,
         config: Optional[Union[Dict, RuntimeEnvConfig]] = None,
         _validate: bool = True,
+        mpi: Optional[Dict] = None,
```

Comment:
We should update a doc, or consider `_mpi`!

Reply:
we'll update the doc. i think it's a feature.